src/api-umbrella/proxy/startup/seed_database.lua

```diff
@@ -150,6 +156,15 @@ local function seed_initial_superusers()
         admin["authentication_token"] = random_token(40)
       end
       admin["updated_at"] = nowMongoDate
+      
```

Comment:
The log is mis-leading and should either be removed or made more concrete. The comment in l160 is als superfluous in my opinion, since thats information present in the git-log